### PR TITLE
Multiple profiles fix

### PR DESCRIPTION
**Pull request by lduarte1991**
@arthurian Take a look at this. Basically now the Edit Course Settings page will only show you unique usernames in the dropdown list which you can select. There's also a textfield that allows you to input an "expected" administrator so even if they haven't logged in you can pre-empt them and add them as admin next time they log in.

I think you should take a look and tell me what the appropriate changes would be for ATG. I'll point out what I think may be areas were we could diverge below. 

Basically this creates a new table where "pending" admins get added. That means allows you to select the appropriate profile since they will be officially added as admins once they log in.

### CODE REVIEW

File: hx_lti_initializer/views.py [hunk 1]
```diff
@@ -213,6 +213,16 @@ def launch_lti(request):
         object_id = get_lti_value(settings.LTI_OBJECT_ID, tool_provider)
         course_id = str(course)
         if set(roles) & set(settings.ADMIN_ROLES):
+            try:
+                userfound = LTICourseAdmin.objects.get(
```
Comment (lduarte1991) [inline hx_lti_initializer/views.py]: @arthurian I would assume that you need a different admin_unique_identifier instead of the username? Or would it work with the username?

File: hx_lti_initializer/views.py [hunk 2]
```diff
@@ -232,6 +253,31 @@ def edit_course(request, id):
...
+            course.save()
+
+            # this will create an item in the database so when the user
+            # that was just added as an admin logs in, they get added
+            # to the list of admins in the course.
+            if len(selected_list) > 0:
+                for name in selected_list:
+                    if str(name.strip()) != "":
+                        try:
+                            new_course_admin = LTICourseAdmin(
```
Comment (lduarte1991) [inline hx_lti_initializer/views.py]: @arthurian Same as above? Is it okay to user username here or should it search for something else?

File: hx_lti_initializer/views.py (hunk 1 above)
Comment (arthurian) [inline hx_lti_initializer/views.py]: @lduarte1991 The `username` should work fine for this actually, since the `username` will be the person's HUID.

I was thinking that maybe we should do email instead (i.e. `lis_person_contact_email_primary`), but the problem with that is people tend to have several harvard email addresses, and there's a chance the person entering the email might put in the wrong one. If we keep it as the `username` (HUID), then it's unambiguous. A little bit of a hassle to enter, but at least it will eliminate that potential for error. Maybe down the line, we can enhance it so that it can be either/or, but for now, let's stick to `username`.

File: hx_lti_initializer/views.py (hunk 2 above)
Comment (arthurian) [inline hx_lti_initializer/views.py]: @lduarte1991 Same as above.

File: hx_lti_initializer/views.py (hunk 1 above)
Comment (lduarte1991) [inline hx_lti_initializer/views.py]: Sounds good and yeah there was quite a bit of pushback from the folks here at Hx in having the learner's emails be in a server outside of the edX platform so I think we would prefer that unless we can make a good case as to why not.

**Comment by arthurian** [comment]
@lduarte1991 Thanks for this feature, it looks good to me. Just to reiterate, `user.username` works for us too. 👍 👍
